BUILD.gn

```diff
@@ -92,6 +92,7 @@ group("gn_all") {
     deps = [
       "//base:base_perftests",
       "//base:base_unittests",
+      "//cobalt",
```

Comment:
We don't want to modify upstream here without gating it with a build flag - the original Chrome code should keep building without modifications. @andrewsavage1 please advice what `if() clause to add at the bottom here.

Reply:
Thanks Andrew. Done.